content/blog/2021-12-15-react-conf-2021-recap.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "React Conf 2021 Recap: React 18 and Beyond"
+author: [jtannady, rickhanlonii]
+---
+
+On December 8th, we hosted our 6th [React Conf](https://conf.reactjs.org/). 50,000 participants registered for the first virtual React conf, which was hosted over a YouTube Live stream and a Discord text server.
+
+**Some quick stats:**
+
+* We streamed the event twice to accommodate community members in different time zones. 6,900 viewers for the Live event and 5,800 viewers for the Replay event ten hours after the Live event.
+* 61,000 people watched the talks after the event on [YouTube](https://www.youtube.com/channel/UC1hOCRBN2mnXgN5reSoO3pQ/videos).
+* 5,700 members participated in the live discussion via our [Discord server](https://discord.gg/JVtWJyagJw).
+* 8 languages were officially supported with professional video captioning and multilingual moderators to accommodate the global React community. React Conf was available in Arabic, Spanish, Hindi, Japanese, Portuguese, Russian, Chinese, and English.
+* A team of moderators were brought in to enforce the [Code of Conduct](https://github.com/facebook/react/blob/main/CODE_OF_CONDUCT.md) at the event.
+
+In previous years, we've used the React Conf stage to deliver industry changing announcements such as [React Native](https://engineering.fb.com/2015/03/26/android/react-native-bringing-modern-web-techniques-to-mobile/) and [React Hooks](https://reactjs.org/docs/hooks-intro.html). This year, we shared our multi-platform vision for React, starting with the release of **React 18** and the **concurrent rendering engine**.
+
+After [piloting React 18](https://engineering.fb.com/2020/05/08/web/facebook-redesign/) with select teams at Meta, and are excited to announce that we're ready to share React 18 with a wider audience. We are releasing the React 18 RC to open source so that members of the React community can begin using the new concurrent rendering engine today.
+
+This release is a milestone of years of collaborative research between the React, React Native, and React Data teams. With this release we're rolling out features that power the [facebook.com](https://www.facebook.com/) site and set the foundation for upcoming features such as **Server Components** and **React Native pre-rendering**.
```

Comment:
If I scan the bolded parts then it’s easy to assume we’re saying that Server Components are going to be in 18. Also I expect that the vast majority of our open source users don’t have context on what RN pre-rendering is so we probably shouldn’t emphasise it here.